docs/_docs/tips.md

```diff
@@ -14,3 +14,34 @@ Try to separate your `@DesignDoc()` declarations into individual files. The
 declarations must be reprocessed and recompiled any time a change is detected in
 it's containing file, so minimizing the amount of code that could change will
 minimize the amount of time spent recompiling your source.
+
+## DCF File Info
+
+When live update downloads a Figma design, the file is saved into a DesignCompose
+file (.dcf) and cached on the device. This file has a version associated with it
+and cannot be used with a different version of DesignCompose. To view the version
+along with some other data in the .dcf file, use the `dcf_info` tool. From the
+root of the DesignCompose repository, run:
+```
+cargo run --bin dcf_info --features=dcf_info <dcf file>
+```
+
+This tool will print the following fields, if it can parse them:
```

Comment:
Would you like to add a short explanation when it can't parse and what happens?

Reply:
Sure, done.